src/bentoml/_internal/cloud/deployment.py

```diff
@@ -550,93 +520,95 @@ def tail_image_builder_logs(stop_spinner: t.Callable[[], None]):
                         break
                     if stop_tail_event.wait(check_interval):
                         return
-                if pod is None:
-                    console.print(
-                        "🚨 [bold red]Can't found the image builder pod.[/bold red]"
-                    )
-                    return
 
                 is_first = True
-                for piece, ws in cloud_rest_client.v2.tail_logs(
+                logs_tailer, close_tail = cloud_rest_client.v2.tail_logs(
                     cluster_name=self.cluster,
                     namespace=self._schema.kube_namespace,
                     pod_name=pod.name,
                     container_name="builder",
-                    stop_event=stop_tail_event,
-                ):
-                    with lock:
-                        ws_session = ws
-                    decoded_bytes = base64.b64decode(piece)
-                    decoded_str = decoded_bytes.decode("utf-8")
+                )
+                stack.callback(close_tail)
+                for piece in logs_tailer:
+                    if stop_tail_event.is_set():
+                        break
+                    try:
+                        decoded_bytes = base64.b64decode(piece)
+                    except binascii.Error:
```

Comment:
This can be deleted because bentocloud will definitely return the content after base64 encoding.